_includes/head.html

```diff
@@ -33,6 +33,7 @@
   {%- if page.sitemap == false or site.GH_ENV == "gh_pages" %}
   <meta name="robots" content="noindex"/>
   {%- endif %}
+  <script async="async" src="//consent.trustarc.com/notice?domain=docker.com&c=teconsent&js=nj&noticeType=bb&gtm=1"></script>
```

Comment:
We should probably make this conditional, so that it doesn't render when building the docs locally, when previewing, or if someone would run the docs elsewhere. (similar to what we do for the google analytics the line below

Reply:
I'll update the PR with this change. Thanks for the input!